<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSegmentDialog</class>
 <widget class="QDialog" name="EditSegmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Segment Information</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SegmentId:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_segmentId">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QComboBox" name="cbLocation">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>#Tracks: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbTracks">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>newer street name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="QLineEdit" name="txtNewerName"/>
   </item>
   <item row="1" column="7" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Points: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLineEdit" name="txtPoints">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Type: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="12" colspan="2">
    <widget class="QComboBox" name="cbRouteType"/>
   </item>
   <item row="1" column="14" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Routes:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="16">
    <widget class="QLineEdit" name="txtRoutes">
     <property name="toolTip">
      <string>Number of routes using this segment.</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>In Service:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QDateEdit" name="dtBegin">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Double-tracked:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="QDateEdit" name="doubleTracked">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Out of Service:  </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="13" colspan="3">
    <widget class="QDateEdit" name="dtEnd">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Direction:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="txtDirection">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Bearing:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5" colspan="2">
    <widget class="QLineEdit" name="txtBearing">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="9" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Length (km): </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="11" colspan="2">
    <widget class="QLineEdit" name="txtKm">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="13" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Length(mi): </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="15" colspan="2">
    <widget class="QLineEdit" name="txtMiles">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="17">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Duplicate Segments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QCheckBox" name="chkDeleteUnused">
     <property name="text">
      <string>Delete unused segments</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="8" colspan="9">
    <widget class="EditSegmentDescr" name="txtDescription">
     <property name="placeholderText">
      <string>enter segment description</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="17">
    <widget class="QLabel" name="lblHelp">
     <property name="styleSheet">
      <string notr="true">color:rgb(246, 97, 81)</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="5" colspan="2">
    <widget class="QPushButton" name="btnRecalc">
     <property name="text">
      <string>Recalculate dates </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditSegmentDescr</class>
   <extends>QLineEdit</extends>
   <header>editsegmentdescr.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbRouteType</tabstop>
  <tabstop>dtBegin</tabstop>
  <tabstop>dtEnd</tabstop>
  <tabstop>txtDirection</tabstop>
  <tabstop>txtBearing</tabstop>
  <tabstop>txtMiles</tabstop>
  <tabstop>txtKm</tabstop>
  <tabstop>txtPoints</tabstop>
  <tabstop>txtRoutes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
